rdmo/core/templates/core/home.html

```diff
@@ -29,7 +29,7 @@
         <div class="header-image header-image-2">
             <img src="{% static 'core/img/header/collection.jpg' %}"></img>
             <p class="attribution">
-                <a href="https://www.flickr.com/photos/bulle_de/9221247815">Nasssammlung (Christopher Bulle)</a> / <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a>
+                <a href="https://www.flickr.com/photos/bulle_de/9221247815">Nasssammlung (Christopher Bulle)</a> / <a href="https://creativecommons.org/licenses/by/2.0/">CC BY 2.0</a> <!-- spellchecker:disable-line -->
```

Comment:
I added a comment to the line containing the name "Bulle". Typos wanted to fix it, as Bullet.